CelebrityStalker.xcodeproj/project.pbxproj

```diff
@@ -18,6 +18,7 @@
 		3FFB72E71BD811C4000E3300 /* CelebrityTabBarController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3FFB72E61BD811C4000E3300 /* CelebrityTabBarController.swift */; settings = {ASSET_TAGS = (); }; };
 		3FFB72E91BD9273B000E3300 /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 3FFB72E81BD9273B000E3300 /* Main.storyboard */; settings = {ASSET_TAGS = (); }; };
 		3FFB72EB1BD93342000E3300 /* EditCelebrityDetailsViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3FFB72EA1BD93342000E3300 /* EditCelebrityDetailsViewController.swift */; settings = {ASSET_TAGS = (); }; };
+		A1C9FAA21BD9369300161C68 /* celebBio.JSON in Resources */ = {isa = PBXBuildFile; fileRef = A1C9FAA11BD9369300161C68 /* celebBio.JSON */; settings = {ASSET_TAGS = (); }; };
```

Comment:
*This is just adding the __JSON file__ to the __PBXBuildFile__ section of the project.*